api/config/config_local.json

```diff
@@ -79,7 +79,7 @@
     "adminIdVerification": "manual-id-verification-requests@fake-research-aou.org"
   },
   "mandrill": {
-    "fromEmail": "donotreply@fake-research-aou.org",
+    "fromEmail": "support@researchallofus.org",
```

Comment:
Does this and perf need to be "fake-research"?

Reply:
I don't think it matters for lower envs